packit_service/worker/jobs.py

```diff
@@ -71,7 +72,7 @@
 logger = logging.getLogger(__name__)
 
 
-MANUAL_EVENTS = [AbstractCommentEvent, CheckRerunEvent]
+MANUAL_EVENTS = [AbstractCommentEvent, AbstractResultEvent, CheckRerunEvent]
```

Comment:
just for the sake of clarity of the code, I would either rename the list to something like `MANUAL_OR_RESULT_EVENTS` or add one more subcondition to the condition you have introduced in the changes before to check whether the event is AbstractResultEvent

Reply:
Makes sense. Pushing changes.